yt_dlp/extractor/tiktok.py

```diff
@@ -911,3 +911,47 @@ def _real_extract(self, url):
         if self.suitable(new_url):  # Prevent infinite loop in case redirect fails
             raise UnsupportedError(new_url)
         return self.url_result(new_url)
+
+
+class TikTokLiveIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?tiktok\.com/@(?P<id>[\w\.-]+)/live'
+    IE_NAME = 'tiktok:live'
+
+    def _real_extract(self, url):
+        uploader_id = self._match_id(url)
```

Comment:
IMO we should try to keep the metadata fields consistent with the other TikTok IEs, in which case the `@username`/`unique_id`/`uniqueId` would instead be `uploader`
```suggestion
        uploader = self._match_id(url)
```